### PR TITLE
Himanshu

### PR DESCRIPTION
**Pull request by HimanshuNandrajog**
Hello

### CODE REVIEW

File: index.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+hello
+
+Hi
```
Comment (Surinder07) [inline index.txt:3]: Please remove this hi

**Comment by AvinashKaur5** [comment]
looks good (Y)

**Review by Dhinesh2206 — COMMENTED** [review]
Change accepted

**Review by Surinder07 — COMMENTED** [review]
Nice work

**Review by Surinder07 — APPROVED** [review]

File: index.txt (hunk 1 above)
Comment (sdkanwar8320TR) [reply index.txt:3]: Approved

Comment (HimanshuNandrajog) [reply index.txt:3]: No

**Comment by Ppkbajwa** [comment]
lets merge

**Pull request merged**